### PR TITLE
feat(inline): Inline calendar

### PR DESCRIPTION
**Pull request by isaacplmann**
Changed the existing `dp-calendar` to `dp-calendar-month`
Added new `dp-calendar` that encapsulates the contents of the popup
There's also a Inline Demo page that shows off the new capabilities.

Todo:
- [x] Add documentation
- [x] Squash commits

### CODE REVIEW

**Comment by isaacplmann** [comment]
Fixes #16

**Comment by vlio20** [comment]
@isaacplmann, please merge with muster and try running pull request again

File: src/app/demo/inline/inline.component.html [hunk 1]
```diff
@@ -0,0 +1,290 @@
+<div>
```
Comment (vlio20) [inline src/app/demo/inline/inline.component.html]: I don't think it is a good idea to have another demo page for the inline component.  There is too much duplicated code here. Let's add it to the current demo page.

**Comment by isaacplmann** [comment]
- Removed the inline demo page and combined it with the main demo page.
- Added documentation to readme
- Fixed some tests

**Comment by vlio20** [comment]
@isaacplmann, thanks for another great PR!

**Comment by isaacplmann** [comment]
You’re welcome.  Thanks for your hard work.  Let me know when the new version is published.

**Comment by vlio20** [comment]
I hope I will publish it tomorrow.

**Pull request merged**